DIMM/v3/_summit.yaml

```diff
@@ -4,7 +4,7 @@ instances:
     controller: astelco
     measurement_validity: 600.0
     config:
-      host: 139.229.170.175
+      host: 139.229.170.176
```

Comment:
This IP is currently mapped to dimm.cp.lsst.org, maybe use the name instead of direct IP?

Reply:
updated with hostname.